app.js

```diff
@@ -1,35 +1,38 @@
 // ℹ️ Gets access to environment variables/settings
 // https://www.npmjs.com/package/dotenv
-require('dotenv/config');
+require("dotenv/config");
 
 // ℹ️ Connects to the database
-require('./db');
+require("./db");
 
 // Handles http requests (express is node js framework)
 // https://www.npmjs.com/package/express
-const express = require('express');
+const express = require("express");
 
 // Handles the handlebars
 // https://www.npmjs.com/package/hbs
-const hbs = require('hbs');
+const hbs = require("hbs");
 
 const app = express();
 
 // ℹ️ This function is getting exported from the config folder. It runs most middlewares
-require('./config')(app);
+require("./config")(app);
 
```

Comment:
You're missing the import of the session configuration file in /config/session.config.js